app/renderer/components/navigation/navigationBar.js

```diff
@@ -242,7 +246,7 @@ class NavigationBar extends ImmutableComponent {
         urlbar={this.props.navbar.get('urlbar')}
         onStop={this.onStop}
         menubarVisible={this.props.menubarVisible}
-        noBorderRadius={!isSourceAboutUrl(this.props.location)}
+        noBorderRadius={(isSourceAboutUrl(this.props.location) && this.isPublisherButtonEnabled) || this.isPublisherButtonEnabled}
```

Comment:
I think you don't need to check aboutUrls since it will return false with `UrlUtil.isHttpOrHttps`. 

`noBorderRadius={this.isPublisherButtonEnabled}` should work as expected